edgelet/edgelet-http-workload/tests/dns-san.rs

```diff
@@ -3,7 +3,7 @@
 // We disable this test on Windows in order to avoid having to add a dependency
 // on OpenSSL.
 #![cfg(not(windows))]
-#![deny(rust_2018_idioms, warnings)]
```

Comment:
do you think it is a good idea to disable this?

Reply:
This was a build error fix, similar to #3465 - edge on k8s branch is still using latest Rust compiler, although this particular file is probably a mistake.